code-env/python/desc.json

```diff
@@ -1,5 +1,5 @@
 {
-  "acceptedPythonInterpreters": [],
+  "acceptedPythonInterpreters": ["PYTHON36"],
   "forceConda": false,
   "installCorePackages": true,
   "installJupyterSupport": false
```

Comment:
Since DSS 12, we need to declare the corePackage version if not it resolve to PANDA023 (and it is known to be incompatible with some python3+ version).
So add:
`"corePackagesSet": "AUTO"` to ensure that the proper code pacakge version will be injected by DSS (depending on the Python interpreter)

Reply:
Thanks. Done !